source/lib/cdk-textract-stack.ts

```diff
@@ -170,35 +198,93 @@ export class CdkTextractStack extends cdk.Stack {
       cloudfrontDocumentsBucketPolicyStatement
     );
 
+    const esLogGroup = new LogGroup(
+      this,
+      this.resourceName("ElasticSearchLogGroup"),
+      {
+        logGroupName: this.resourceName("ElasticSearchLogGroup")
+      }
+    );
+
+    // Elasticsearch
+    let elasticSearch;
+
     const esEncryptionKey = new kms.Key(this, "esEncryptionKey", {
       enableKeyRotation: true
     });
-    // Elasticsearch
-    const elasticSearch = new es.CfnDomain(
-      this,
-      this.resourceName("ElasticSearchCluster"),
-      {
-        elasticsearchVersion: "6.5",
-        elasticsearchClusterConfig: {
-          instanceType: "m5.large.elasticsearch"
-        },
-        ebsOptions: {
-          ebsEnabled: true,
-          volumeSize: 20,
-          volumeType: "gp2"
-        },
-        encryptionAtRestOptions: {
-          enabled: true,
-          kmsKeyId: esEncryptionKey.keyId
+
+    if (!props.isCICDDeploy) {
```

Comment:
Just curious, is there a reason not to apply the same configuration for both CICD and non-CICD deploy.
I get that one advantage is additional pricing due to the node-to-node encryption and logs, but is there any other reason for this if-else statement?

Reply:
the only reason is to avoid additional pricing, which also includes multi-az deployment and will, in future, also include creation of a VPC.